### PR TITLE
Shuffleboardben

### PR DESCRIPTION
**Pull request by Juzhu2**
Changes gyro by x number of degrees

Button: X

### CODE REVIEW

**Review by Juzhu2 — COMMENTED** [review]
bennables approved

**Review by Bennables — APPROVED** [review]
approved.

File: src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java [hunk 1]
```diff
@@ -470,4 +470,8 @@ public static PPSwerveControllerCommand followTrajectory(
         Constants.AutoConstants.m_thetaController,
         d::setModuleStates);
   }
+
+  public Object getCurrentAngle() {
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java]: remove unfinished stuff

File: src/main/java/frc/robot/commands/AutoChooser.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/commands/AutoChooser.java]: remove auto-chooser from this branch

**Review by RobertTheBlair — CHANGES_REQUESTED** [review]
need cleanup of unused items

**Review by RobertTheBlair — APPROVED** [review]

**Pull request merged**